scripts/prometheus.yml

```diff
@@ -0,0 +1,10 @@
+# When the Thanos remote-write-receive component is started,
```

Comment:
Hm.. can we add this to `remote-write-receive` command description instead?  Might be easier to discover